src/main/resources/com/puppycrawl/tools/checkstyle/checks/javadoc/messages.properties

```diff
@@ -13,6 +13,7 @@ javadoc.invalidInheritDoc=Invalid use of the '{'@inheritDoc'}' tag.
 javadoc.legacyPackageHtml=Legacy package.html file should be removed.
 javadoc.missed.html.close=Javadoc comment at column {0} has parse error. Missed HTML close tag ''{1}''. Sometimes it means that close tag missed for one of previous tags.
 javadoc.missing=Missing a Javadoc comment.
+javadoc.missing.asterisk=Missing leading asterisk
```

Comment:
As for me, `Javadoc line is missing leading asterisk.` sounds better.
Please also mind the dot at the end.

Reply:
Done